src/Raven.Client/Documents/Queries/Timings/QueryTimings.cs

```diff
@@ -10,6 +11,8 @@ public sealed class QueryTimings : IFillFromBlittableJson, IDynamicJson
         public long DurationInMs { get; set; }
 
         public IDictionary<string, QueryTimings> Timings { get; set; }
+        
+        public object QueryPlan { get; set; }
```

Comment:
object?

Reply:
Changed to `IDynamicJson`, we really don't want to have a limit of what we can put here.

Reply:
Done in https://github.com/ravendb/ravendb/pull/17176/commits/04d16c6516dd9974c952589f8d049b5907c90e0d